schema/google/showcase/v1beta1/echo.proto

```diff
@@ -96,7 +96,7 @@ service Echo {
 }
 
 // The request message used for the Echo, Collect and Chat methods. If content
-// is set in this message then the request will succeed. If a status is
+// is set in this message then the request will succeed.
```

Comment:
Can you please instead fix it to say: `If status is set in this message then the status will be returned as an error.`

Reply:
Done.